src/composables/subgraph/useSubgraphQuery2.ts

```diff
@@ -0,0 +1,39 @@
+import { useQuery, UseQueryOptions } from 'vue-query';
+import { reactive, Ref } from 'vue';
+import axios from 'axios';
+import { configService } from '@/services/config/config.service';
+
+export const subgraphs = {
+  gauge: configService.network.subgraphs.gauge,
+  balancer: configService.network.subgraph
+};
+
+export default function useSubgraphQuery2<T>(
+  subgraphUrl: string,
```

Comment:
yeah I think having this as an actual URL is a pretty good idea, given the fact that we might need to query subgraphs apart from above for some use cases